### PR TITLE
Fix OWNERS files with allowed keywords

### PR DESCRIPTION
**Pull request by Garima-Negi**
This PR is in reference to issue [https://github.com/kubernetes/community/issues/6334](https://github.com/kubernetes/community/issues/6334)

### CODE REVIEW

File: KUBERNETES_CSI_OWNERS_ALIASES [hunk 1]
```diff
@@ -4,7 +4,7 @@ aliases:
 
   # SIG-Storage chairs and leads should always have approval rights in all repos.
   # Others may be added as needed here or in each repo.
-  kubernetes-csi-approvers:
+  approvers:
```
Comment (dims) [inline KUBERNETES_CSI_OWNERS_ALIASES:7]: you can leave the `kubernetes-csi-` here. this is an alias.

File: KUBERNETES_CSI_OWNERS_ALIASES [hunk 2]
```diff
@@ -16,7 +16,7 @@ aliases:
   #
   # Reviewers are encouraged to set the "Busy" flag in their GitHub status
   # when they are temporarily unable to review PRs.
-  kubernetes-csi-reviewers:
+  reviewers:
```
Comment (dims) [inline KUBERNETES_CSI_OWNERS_ALIASES:19]: you can leave the `kubernetes-csi-` here. this is an alias.

File: KUBERNETES_CSI_OWNERS_ALIASES [hunk 3]
```diff
@@ -31,15 +31,7 @@ aliases:
 
 # This documents who previously contributed to Kubernetes-CSI
 # as approver.
-emeritus_approver:
+emeritus_approvers:
```
Comment (dims) [inline KUBERNETES_CSI_OWNERS_ALIASES:34]: yep this is good

File: KUBERNETES_CSI_OWNERS_ALIASES (hunk 1 above)
Comment (Garima-Negi) [reply KUBERNETES_CSI_OWNERS_ALIASES]: done

File: KUBERNETES_CSI_OWNERS_ALIASES (hunk 2 above)
Comment (Garima-Negi) [reply KUBERNETES_CSI_OWNERS_ALIASES]: done

File: KUBERNETES_CSI_OWNERS_ALIASES [hunk 4]
```diff
@@ -31,15 +31,7 @@ aliases:
...
 - lpabon
 - sbezverk
 - vladimirvivien
-
-# This documents who previously contributed to Kubernetes-CSI
-# as reviewer.
-emeritus_reviewer:
```
Comment (xing-yang) [inline KUBERNETES_CSI_OWNERS_ALIASES:41]: The reviewers list should not be removed as it is not exactly the same as the approvers list.

Comment (dims) [reply KUBERNETES_CSI_OWNERS_ALIASES:41]: @xing-yang then you can track that using comments or something? `emeritus_reviewer` is not a supported yaml section in our tooling.

Comment (xing-yang) [reply KUBERNETES_CSI_OWNERS_ALIASES:41]: Do we have emeritus_reviewers?

Comment (dims) [reply KUBERNETES_CSI_OWNERS_ALIASES:41]: @xing-yang nope. only thing allowed is `emeritus_approvers` / `approvers` / `reviewers`

Comment (xing-yang) [reply KUBERNETES_CSI_OWNERS_ALIASES:41]: I see.

**Comment by xing-yang** [comment]
@Garima-Negi can you squash the commits?

**Comment by Garima-Negi** [comment]
@xing-yang @dims squashed commits as required, this is ready for merge

**Comment by pohly** [comment]
> squashed commits as required, this is ready for merge

I still see a second commit?

**Comment by Garima-Negi** [comment]
> > squashed commits as required, this is ready for merge
> 
> I still see a second commit?

You're right, I missed one commit. Fixed that now, please take a look.

**Comment by pohly** [comment]
You might want to read up on [good practices for commit messages](https://www.kubernetes.dev/docs/guide/pull-requests/#commit-message-guidelines) for future contributions, but for now this is okay.

/lgtm
/approve

**Pull request merged**